src/zenml/cli/utils.py

```diff
@@ -1230,3 +1230,42 @@ def wrapper(function: F) -> F:
         return wrapper(func)
 
     return inner_decorator
+
+
+def get_installed_local_packages() -> Dict[str, str]:
+    """Get a dictionary of all locally installed packages.
+
+    Returns:
+        A dictionary of all locally installed packages and their versions.
+    """
+    import pkg_resources
+
+    return {pkg.key: pkg.version for pkg in pkg_resources.working_set}
+
+
+def get_package_information(package_names: List[str]) -> Dict[str, str]:
```

Comment:
Tiny nit, but this and the function above are very similar and we could probably unify them? E.g. make the parameter an optional `include` list, and if given only those packages get returned. If left empty, all packages get returned

Reply:
Done